images/source/Dockerfile

```diff
@@ -8,4 +8,4 @@
 FROM centos:7
 
 ADD origin-local-release.repo /etc/yum.repos.d/
-ADD centos-paas-sig-openshift-future.repo /etc/yum.repos.d/
+ADD centos-paas-sig-openshift-origin36.repo /etc/yum.repos.d/
```

Comment:
There is no need for rebasing since the existing origin36 changes in master are in the node image. I figured that it should change for the source image, is this not what we want?